src/main/webapp/Event.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Events format class
+ */
+export default class Event {
+    constructor(title, location, startingTime, endingTime) {
+        this.title = title;        
+        this.location = location;        
+        this.startingTime = startingTime;        
+        this.endingTime = endingTime;        
+    }
+}
```

Comment:
I am not sure if that's the right approach to have the Event object. The LocalStorage only supports strings. So, the best way to add stuff is to have them converted to strings using `JSON.parse()`.

This allows structures like objects, numbers, arrays, etc. to be converted into a string (in JSON format) and to make them LocalStorage frendly. However, I have no idea whether it can convert classes like this (any thoughts @semenovm @maxwbuckley ?). I guess, we can try like this for now and, if that doesn't work out, we can switch to JSON?

Reply:
AFAIK JSON.stringify will handle it fine, and it easy to test if not. @RemusNeatu please test it.

Reply:
I tested it and the JSON.stringify works just fine for the class I created. If I call JSON.stringify(event), where event is an Event object, the result is a JSON with the format {title: "...", location: "..." ... }. When we add the event in the LocalStorage we just use JSON.stringify.